shell/browser/browser_win.cc

```diff
@@ -512,33 +589,87 @@ bool Browser::SetBadgeCount(int count) {
 }
 
 void Browser::SetLoginItemSettings(LoginItemSettings settings) {
-  base::string16 keyPath = L"Software\\Microsoft\\Windows\\CurrentVersion\\Run";
-  base::win::RegKey key(HKEY_CURRENT_USER, keyPath.c_str(), KEY_ALL_ACCESS);
+  base::string16 key_path =
+      L"Software\\Microsoft\\Windows\\CurrentVersion\\Run";
+  base::win::RegKey key(HKEY_CURRENT_USER, key_path.c_str(),
+                        KEY_ALL_ACCESS | KEY_WOW64_64KEY);
```

Comment:
I wonder if we should use the WOW64 key here. We don't wanna force it to go there. Lets check  with the API that we do therigth thing like this:
32bit app on 32bit OS -> write to normal reg path
32bit app on 64bit OS -> write to WOW6432Node reg path
32bit app on 64bit OS -> write to normal reg path